app/containers/Settings/Settings.jsx

```diff
@@ -257,18 +258,19 @@ export default class Settings extends Component<Props, State> {
               noBorderBottom
               title="THEME"
             >
-              <Tooltip
-                className={styles.settingsSelectContainer}
-                title="Coming Soon"
-                position="bottom"
-              >
+              <div className={styles.settingsSelectContainer}>
                 <StyledReactSelect
                   settingsSelect
-                  isDisabled
+                  onChange={this.updateThemeSettings}
+                  isSearchable={false}
                   transparent
+                  options={[
+                    { value: 'Light', label: 'Light' },
+                    { value: 'Dark', label: 'Dark' }
+                  ]}
```

Comment:
For consistency's sake should these be mapped constants from `core/constants` like currency and explorer options are?